frontend/shared/src/components/modal/Modal.tsx

```diff
@@ -65,7 +67,7 @@ const Modal: React.FC<ModalProps> = ({
               onClick={handleToggle}
               data-testid="cancel"
             >
-              {t('common:applications.actions.close')}
+              {cancelButtonLabel}
```

Comment:
It's not absolutely necessary to parameterize every text in shared components, if just use different texts in different application. So if you use shared Modal component, just bring translation with using the same key in your service.

Pros: less properties for shared component
Cons: have to use exactly the same key.